src/commissaire/bus/__init__.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, message, code=0, data={}):
         :param data: Additional error data.
         :type data: dict
         """
-        super(RemoteProcedureCallError, self).__init__(message, code, data)
+        super().__init__(message, code, data)
```

Comment:
Would that typically be called out in `requirements.txt` or `setup.py`, or just documented in something like `README.md`?